### PR TITLE
Create endpoint to return 3 phrases

### PR DESCRIPTION
**Pull request by swmschmidt**

### CODE REVIEW

File: src/main/kotlin/com/game/catbackend/api/PhraseAPI.kt [hunk 1]
```diff
@@ -26,5 +26,10 @@ class PhraseAPI(val phraseService: PhraseService) {
         return phraseService.getAll()
     }
 
+    @GetMapping("/random")
+    fun getLast3Phrases(): List<Phrase> {
+        return phraseService.getAll().takeLast(3)
```
Comment (Orlando-Silva) [inline src/main/kotlin/com/game/catbackend/api/PhraseAPI.kt]: Although correct, this shouldn't be the way.

Mainly because of separation of concerns. Ideally, every layer has its responsabilities. The layer we're currently in (`API`/`Controller`) layer should not have the responsability of holding business logic (in this case, take last 3 could be considered a business logic).

Also, you're reusing `phraseService.getAll()` method. Which, in most cases it is a good thing, however, in this case, we should create a new method in `phraseService`. The reasoning behind it is that if `getAll`, for some reason, change its logics, we might not want this endpoint (`/random`) to be affected by it (eg: say `getAll` now has to filter only active entries and `/random` should not. If both are using the same method, the logic will be changed in `phraseService.getAll()` and might cause a unwanted side effect on `/random`).

TLDR: Ideally, we should create a new method in phraseService and call this method here.

```suggestion
        return phraseService.takeLastThree()
```

**Review by Orlando-Silva — APPROVED** [review]
